runtime/near-vm-logic/src/context.rs

```diff
@@ -68,3 +68,26 @@ impl VMContext {
         self.view_config.is_some()
     }
 }
+
+impl Default for VMContext {
+    fn default() -> Self {
+        VMContext {
```

Comment:
I am not sure whether implementing default is a good idea. It looks like this implementation is used solely for testing. In that case we should probably implement a standalone function `VMContext::test`

Reply:
+1

In addition to what Bowen said, this is also the case where the duplication is superficial -- nothing breaks if the two usages would use different contracts. The problem with deduping here is that, if in the future it turns out that you *do* want to have different contexts, untangling that would be difficult. 

It's easier to notice duplication when to notice that some things are merged while they need to be separate. 

I think it's better not to extract this at all (but no strong opinion -- adding an `put fn default_test_context` looks fine by me!)

Reply:
Okay, that seems to be bigger issue than dealing with different `VMContext`. Thank you for reviewing!